bot/config/default.json

```diff
@@ -1,3 +1,4 @@
 {
-	"access_token": "579386051:AAGs0r6ZWS6E5LHXBQqx55Ga2OCyd-_pdPs" 
-}
\ No newline at end of file
+	"access_token": "579386051:AAGs0r6ZWS6E5LHXBQqx55Ga2OCyd-_pdPs",
+	"api_token": "ca92eab5d887400bb5e7a48a4831d838"
```

Comment:
I think we should hide the API keys. I think we should
1. make a `bot/config/default.example.json` file that includes the same fields but whateverrandomarbitrary values
1. add this file to .gitignore